### PR TITLE
Update common_voice_privacy_notice.md

### PR DESCRIPTION
**Pull request by smtiernan**
Updates to reflect spontaneous speech project

### CODE REVIEW

File: en/common_voice_privacy_notice.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ When Mozilla (that’s us), receives information from you, our [Mozilla Privacy
 
 * **Account data.** You do not need to create an account to use Common Voice. If you decide to create an account, we receive your username and avatar, if you submit one. Your email address is associated with your demographic and interaction data but is not shared to the public. We display leaderboards showing the number of recordings users make. You have the option whether or not you wish to appear on the leaderboards. You can delete your account at any time and your username and email will be removed.
 
+* **Email.** If you participate in the Alpha or Beta testing for the Common Voice Spontaneous Speech project, you must provide your email address so that we can send you a link to log in to participate in the Common Voice Spontaneous Speech project. At any time, you may create a full account (as set forth above). If you do not create a full account upon completion of the Alpha or Beta testing, we will delete your email address within two weeks after the completion of each testing program. Certain types of participation in Beta testing (e.g. for ultra low resource languages) may not require an email log-in.
```
Comment (peiying2) [inline en/common_voice_privacy_notice.md:13]: should it be `login` (without the hyphen) instead?

**Review by peiying2 — COMMENTED** [review]
The rest looks good. Just one minor comment

File: en/common_voice_privacy_notice.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Common Voice Privacy Notice 
 
-Effective August 15, 2019 {: datetime="2019-08-15" }
+Effective June 27, 2024
```
Comment (flodolo) [inline en/common_voice_privacy_notice.md:3]:
```suggestion
Effective June 27, 2024 {: datetime="2024-06-27" }
```

You need to add back the `datetime` part.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: en/common_voice_privacy_notice.md (hunk 1 above)
Comment (smtiernan) [reply en/common_voice_privacy_notice.md]: Yes, I changed it. Thank you.

**Review by peiying2 — APPROVED** [review]

**Comment by peiying2** [comment]
@flodolo when you get a chance, please review and approve. It's blocking the merge.

**Pull request merged**